docs/source/seth_developers_guide/getting_started.rst

```diff
@@ -105,13 +99,13 @@ the key file must be the same as the alias the account is imported as.
 Now we are ready to set up the account on the network. To do this, we need to
 use the ``seth`` command. From the prompt where you generated the key, run::
 
-    # seth account import alias.pem alias
+    # seth account import alias.pem myalias
```

Comment:
Currently the PEM filename and the alias *must* be the same, so perhaps change to `{alias}.pem {alias}` as in previous examples.

Reply:
Looking at the code it seems to me that `seth account import {key-file} {alias}` writes a file called `{alias}.pem` but it doesnt matter what the `{key-file}` is called. My goal was to make the example commands easy to copy-paste in, so I'd like to avoid the brackets. But it might be better if the example key file it generates is something like "key-file.pem"